lib/service/signals.go

```diff
@@ -184,13 +185,16 @@ func (process *TeleportProcess) closeImportedDescriptors(prefix string) error {
 	defer process.Unlock()
 
 	var errors []error
-	for i := range process.importedDescriptors {
-		d := process.importedDescriptors[i]
+	openDescriptors := process.importedDescriptors[:0]
```

Comment:
I must say that this quite tricky I needed at lest  2min to figure out why openDescriptors is created from `importedDescriptors[:0]`. 
Can we use an additional slice for that instead of reusing the space from `importedDescriptors` to keep unclosed descriptor and focus on Readability ?